src/data/matrix/basic.lean

```diff
@@ -964,6 +964,9 @@ lemma transpose_reindex (eₘ : m ≃ l) (eₙ : n ≃ o) (M : matrix m n α) :
   (reindex eₘ eₙ M)ᵀ = (reindex eₙ eₘ Mᵀ) :=
 rfl
 
+@[simp] lemma reindex_one [semiring α] [decidable_eq m] [decidable_eq l] (eₘ : m ≃ l) :
+  (reindex eₘ eₘ (1 : matrix m m α)) = 1 := by simp
+
```

Comment:
I have at any rate wiped off the `reindex_one`, as the `minor` could close the goal I was interested in, in `linear_algebra/matrix/reindex.lean`.